scripts/release/socket.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
```

Comment:
maybe add `set -e` ?

Reply:
also, in the case of this script you also want set the option so that and error on one pipe would break the entire command.

Reply:
I'll add the `set -e`.  I'm afraid I didn't understand your second comment.